client/my-sites/people/viewers-list/index.jsx

```diff
@@ -43,6 +31,21 @@ const ViewersList = ( { site, label } ) => {
 			);
 		}
 	}, [ dispatch, error, refetch, translate ] );
```

Comment:
The notice and the button labels should be translated. Right now, the `translate` function is not used at all in this hook.

Some typos and consistency issues worth fixing:
- say "error retrieving viewers" instead of "error retrieving viewer"
- put a period after "Try again." The `useErrorNotice` in team list does that

Reply:
Oops, that's not intended. Fix is here https://github.com/Automattic/wp-calypso/pull/50890/commits/2e45107693784786d36c117d862cc9965041fa59 – regarding the latter, I'd rather remove the period from the `useErrorNotice` in TeamList tbh.